features/react_native.feature

```diff
@@ -126,22 +126,21 @@ Scenario: Source maps are uploaded in an app using Hermes
 Scenario: Plugin handles server failure gracefully
     When I set the HTTP status code to 500
     And I run the script "features/scripts/manual_upload_react_native.sh" synchronously
-    And I wait to receive 5 requests
+    And I wait to receive 5 builds
     And 5 requests are valid for the JS source map API and match the following:
         | appVersionCode | appVersion | overwrite | dev   |
         | 5              | 2.45.beta  | false     | false |
         | 5              | 2.45.beta  | false     | false |
         | 5              | 2.45.beta  | false     | false |
         | 5              | 2.45.beta  | false     | false |
         | 5              | 2.45.beta  | false     | false |
-    And the exit code equals 0
```

Comment:
Is there a reason for the removal of this assertion?

Reply:
Yes, I was going to make it implicit to running the react-native and NDK scripts, which I hadn't pushed up.  It's now been added to the branch.